README.md

```diff
@@ -39,21 +38,11 @@ Simply add it to the folder containing your component. The browser test command
 
 ## Environment
 
-We haven't yet added any utilities to help load environment variables; you can do it yourself by typing in the following in your terminal:
+This project requires several environment variables be configured before it is able to run. It uses a library called [`habitat`](https://github.com/brianloveswords/habitat) to load configuration from a `.env` file.
```

Comment:
ya it's true. i think this gives an ok overview- i wouldn't want to go into too much more detail, unless you want to turn the above into a 10 word summary?